invenio_users_resources/entity_resolvers.py

```diff
@@ -65,7 +65,7 @@ def pick_resolved_fields(self, identity, resolved_dict):
             "avatar"
         ]
         # TODO trying to recreate the user item here, there must be a better way!
-        links = resolved_dict.get("links")
+        links = resolved_dict.get("links") or {}
```

Comment:
in case on `ghost_record`, there is no `links` in the dict, but it still needs an avatar